components/onboard/youtube-video.js

```diff
@@ -1,10 +1,15 @@
 const YoutubeVideo = ({
-  width=560,
-  height=315,
   'youtube-id': youtubeID = 'dQw4w9WgXcQ',
+  list=null,
   title="YouTube video player",
+  width=null,
+  height=null,
```

Comment:
```suggestion
  list,
  title = "YouTube video player",
  width,
  height,
```

Max.. you write too much Ruby

Reply:
called out– I actually typed `nil` on these lines before correcting myself.

Reply:
😂 I know the feeling